src/test/scala/is/hail/variant/vsm/VSMSuite.scala

```diff
@@ -448,4 +450,19 @@ class VSMSuite extends SparkSuite {
     intercept[HailException](vds.reorderSamples(newOrder))
     intercept[HailException](vds.reorderSamples(vds.sampleIds.toArray ++ Array[Annotation]("foo", "bar")))
   }
+  
+  @Test def testWriteBlockMatrix() {
+    val dirname = tmpDir.createTempFile()
+    
+    for {
+      numSlices <- Seq(1, 2, 4, 9, 11)
+      blockSize <- Seq(1, 2, 3, 4, 6, 7, 9, 10)
+    } {
+      val vsm = hc.baldingNicholsModel(1, 6, 9, Some(numSlices))
+
+      vsm.writeBlockMatrix(dirname, "g.GT.gt", blockSize)
+      
+      assert(BlockMatrix.read(hc, dirname).toLocalMatrix() === TestUtils.vdsToMatrixDouble(vsm).t)
```

Comment:
This isn't going to fly since the first thing we're going to do after this is delete `vdsToMatrixDouble`.